python/feathub/processors/local/ast_evaluator/ast_evaluator.py

```diff
@@ -0,0 +1,144 @@
+#  Copyright 2022 The Feathub Authors
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+from datetime import datetime
+from typing import Any, Dict, Optional
+
+from feathub.common.exceptions import FeathubException, FeathubExpressionException
+from feathub.dsl.abstract_ast_evaluator import AbstractAstEvaluator
+from feathub.dsl.ast import (
+    ArgListNode,
+    VariableNode,
+    FuncCallOp,
+    ValueNode,
+    CompareOp,
+    UminusOp,
+    BinaryOp,
+    LogicalOp,
+    CastOp,
+    GroupNode,
+)
+from feathub.processors.local.ast_evaluator.functions import get_predefined_function
+
+_TRUE_STRINGS = ("t", "true", "y", "yes", "1")
+_FALSE_STRINGS = ("f", "false", "n", "no", "0")
+
+
+class AstEvaluator(AbstractAstEvaluator):
```

Comment:
Would it be more readable to rename this class as `LocalAstEvaluator`? This would also be more consistent with the way we name `FlinkProcessor` and `LocalProcessor`.

If we make this change, please rename test class as appropriate.